lib/krikri/enrichments/convert_to_sentence_case.rb

```diff
@@ -0,0 +1,20 @@
+module Krikri::Enrichments
+  ##
+  # Converts a string to sentence case.
+  #
+  # @example
+  #
+  #   string = 'this is a sentence about Moomins. this is another about Snorks.'
+  #   Krikri::Enrichments::ConvertToSentenceCase.enrich_value(string)
+  #   # => 'This is a sentence about moomins. This is another about snorks.'
```

Comment:
![](http://vignette2.wikia.nocookie.net/moomin/images/1/1c/Snorkfrc3b8ken-vind-i-hc3a5ret-gif-kopi.gif/revision/latest/scale-to-width/185)